src/lpc.jl

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
So when you say separate functionality, do you mean that you want an `LPCBurg` module and an `LPCLevinson` module each containing a single function, then `@reexport` those functions into the main `DSP` module?